index.md

```diff
@@ -48,11 +30,11 @@ We're located on the 11th floor. Park in the garage and take the elevator to the
 Email: <info@paircolumbus.org>
 
 ### Friends & Links
-- @paircolumbus
+- [GitHub/paircolumbus](https://github.com/paircolumbus)
```

Comment:
@ anything is often recognized as a twitter account. i liked the prominence of the old 'view on github' link. it probably makes more sense to include a reference to GitHub near the top of index. 

we can DRY up challenges as well, since submission instructions are now on the guide.

```
- New Members: Start Here
- Track your Challenge Progress
- Challenge Guide
- View our GitHub Repositories
```